skyreader/plot/plot.py

```diff
@@ -95,7 +97,11 @@ def create_plot(
         (
             grid_value, grid_ra, grid_dec, equatorial_map
         ) = extract_map(
-            result, llh_map, angular_error_floor, remove_min_val=not llh_map
+            result,
+            llh_map,
+            angular_error_floor,
+            remove_min_val=not llh_map,
+            return_uniqs=False,
         )
```

Comment:
Are you using `black` to format this? I think `grid_value, grid_ra, grid_dec, equatorial_map = extract_map(` would fit within 88 chars.

Reply:
I am using flake8, but I think I changed because it looked long to me, but indeed it isn't too long. Changed it.